zerver/openapi/zulip.yaml

```diff
@@ -4310,6 +4310,20 @@ paths:
                                             **Changes**: New in Zulip 10.0 (feature level 298). Previously
                                             `user_group_edit_policy` field used to control the permission
                                             to create user groups.
+                                    can_manage_all_groups:
+                                      allOf:
+                                        - $ref: "#/components/schemas/GroupSettingValue"
+                                        - description: |
+                                            A [group-setting value](/api/group-setting-values) defining
+                                            the set of users who have permission to manage user
+                                            groups in this organization.
```

Comment:
"to administer all existing user groups in this organization" would be clearer language about what precisely the setting allows one to do.

Reply:
Made this tweak, and also copied the API documentation in place such that one can interpret older values correctly, including a mention of the upcoming change in #25942, just to simplify how much we need to refactor these docs.